.nais/default.json

```diff
@@ -0,0 +1,3 @@
+{
+    "namespace": "default"
```

Comment:
Trenger ikke denne filen siden vi bare er i ett name space. Så  tenker vi bare kan hardkode "default" for qa også.